packages/react/src/tabs/indicator/TabsIndicator.tsx

```diff
@@ -94,12 +203,12 @@ export const TabsIndicator = React.forwardRef(function TabIndicator(
 
 export namespace TabsIndicator {
   export interface State extends TabsRoot.State {
-    selectedTabPosition: ActiveTabPosition | null;
-    selectedTabSize: ActiveTabSize | null;
+    selectedTabPosition: TabPosition | null;
```

Comment:
The TabPosition and TabSize types should be publicly exported as they can be used in a render function